src/fastify-plugins/maps/index.js

```diff
@@ -76,7 +81,19 @@ async function routes(fastify, opts) {
     async (req, rep) => {
       const serverAddress = await getFastifyServerAddress(req.server.server)
 
-      // 1. Attempt to get "default" local static map's style.json
+      // 1. Attempt to use the styled map package
+      {
+        const style = await fastify.comapeoSmp.getStyle().catch(() => {
+          fastify.log.warn('Cannot read styled map package archive')
+          return null
+        })
+
+        if (style) {
+          return style
+        }
+      }
```

Comment:
I made the assumption that we want to give highest precedence to an SMP if possible